src/getHangulAcronym.ts

```diff
@@ -0,0 +1,20 @@
+import { isHangul } from './_internal/hangul';
+
+/**
+ *
+ * @param getHangulAcronym
+ * @description
+ * 한글 문장을 입력받아서, 해당 한글 문장의 초성을을 리턴해줍니다.
+ * 한글 문장이 아닌, 문장은 취급하지않습니다. 추가로 한글 문장 + 영어 문장의 경우에도 취급하지않습니다.
+ */
+export function getHangulAcronym(text: string) {
```

Comment:
파라미터를 조금 더 명확하게 해주면 좋겠습니다.
그래야, 한글만 이 함수에 넣어야한다는것을 조금이라도 더 알 것 같아요

```suggestion
export function getHangulAcronym(hangul: string) {
```